src/dsp_tools/schemas/project.json

```diff
@@ -96,7 +98,8 @@
                 "TextValue",
                 "TimeValue",
                 "UriValue"
-            ]
+            ],
+            "default": "TextValue"
```

Comment:
is it a reasonable assumption that `TextValue` is the default value type one wants to define? Wherever there isn't a reasonable default, we probably shouldn't add anything, otherwise users have to delete the default more often than leaving it in.